thirdparty/KDDockWidgets/src/private/multisplitter/Item.cpp

```diff
@@ -714,6 +714,7 @@ void Item::setGeometry(QRect rect)
 
 void Item::dumpLayout(int level)
 {
+    return;
```

Comment:
have you forgotten that?

Reply:
no, I'm bored with the output of this dump when the application starts, the console is heavily littered